### ISSUE TITLE
TypeError: domain.enter is not a function when using bugsnag (Node v10.15.0)

### ISSUE THREAD

**Issue by jeremyjohn** [xmppjs/xmpp.js#627]
```
[bugsnag] Reported an unhandled rejection…
TypeError: domain.enter is not a function
    at Client.EventEmitter.emit (domain.js:458:10)
    at Client._status (C:\Projects\my_project\node_modules\@xmpp\connection\index.js:159:10)
    at Client.connect (C:\Projects\my_project\node_modules\@xmpp\connection\index.js:198:10)
    at Client.connect (C:\Projects\my_project\node_modules\@xmpp\client-core\lib\Client.js:42:18)
    at Client.connect (C:\Projects\my_project\node_modules\@xmpp\resolve\index.js:62:23)
    at Client.start (C:\Projects\my_project\node_modules\@xmpp\connection\index.js:184:16)
    at Object.<anonymous> (C:\Projects\my_project\my_project.js:28:6)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
```
I'm using node v10.15.0 and bugsnag for this project but bugsnag is detecing an unhandled rejection. If i disable bugsnag the error goes away but the point is i want to use bugsnag to detect all unhandled errors and some handler errors which is what it's doing.

**Comment by trampi** [comment]
@sonnyp could you give an estimate for a bugfix release?

**Comment by sonnyp** [comment]
fixed in [v0.7.0](https://github.com/xmppjs/xmpp.js/releases/tag/v0.7.0)
